### DISCUSSION TITLE
Is it possible to import a new window in notebook?   Python tkinter

### DISCUSSION THREAD

**Discussion by baker0808** [israel-dryer/ttkbootstrap#287] [Q&A]
As I had make two stand only program by tkinter,
and new I need to let them run in one program and select by tabs.

So is it possible to import a new window in notebook or import to Frame?
Thanks.

**Comment by israel-dryer** [comment]
I do not understand your question. But, I will guess and let me know if I'm wrong.

It sounds like you have two different windows and you want them in a single window on multiple tabs. If this is true, then you can take everything from the first window and put it into a frame instead of a window. Then, put this frame into the notebook.
